lib/clusters/thermostat.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const Cluster = require('../Cluster');
+const { ZCLDataTypes } = require('../zclTypes');
+
+const ATTRIBUTES = {
+  localTemperature: { id: 0, type: ZCLDataTypes.int16 },
+  outdoorTemperature: { id: 1, type: ZCLDataTypes.int16 },
+  ocupancy: { id: 2, type: ZCLDataTypes.map8 },
+  absMinHeatSetpointLimit: { id: 3, type: ZCLDataTypes.int16 },
+  absMaxHeatSetpointLimit: { id: 4, type: ZCLDataTypes.int16 },
+  absMinCoolSetpointLimit: { id: 5, type: ZCLDataTypes.int16 },
+  absMaxCoolSetpointLimit: { id: 6, type: ZCLDataTypes.int16 },
+  pICoolingDemand: { id: 7, type: ZCLDataTypes.uint8 },
+  pIHeatingDemand: { id: 8, type: ZCLDataTypes.uint8 },
+  
+  localTemperatureCalibration: { id: 16, type: ZCLDataTypes.int8 },
+  occupiedCoolingSetpoint: { id: 17, type: ZCLDataTypes.int16 },
+  occupiedHeatingSetpoint: { id: 18, type: ZCLDataTypes.int16 },
+  unoccupiedCoolingSetpoint: { id: 19, type: ZCLDataTypes.int16 },
+  unoccupiedHeatingSetpoint: { id: 20, type: ZCLDataTypes.int16 },
+  minHeatSetpointLimit: { id: 21, type: ZCLDataTypes.int16 },
+  maxHeatSetpointLimit: { id: 22, type: ZCLDataTypes.int16 },
+  minCoolSetpointLimit: { id: 23, type: ZCLDataTypes.int16 },
+  maxCoolSetpointLimit: { id: 24, type: ZCLDataTypes.int16 },
+  minSetpointDeadBand: { id: 25, type: ZCLDataTypes.int8 },
+  remoteSensing: { id: 26, type: ZCLDataTypes.map8 },
+  controlSequenceOfOperation: { id: 27, type: ZCLDataTypes.enum8 },
+  systemMode: { id: 28, type: ZCLDataTypes.enum8 },
```

Comment:
You need to provide the right values for the `enum8` data type, check other clusters for how this works.